final_task/setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import find_namespace_packages, setup
+
+try:
+    from pip._internal.req import parse_requirements
+except ImportError:
+    from pip.req import parse_requirements
+
+with open('final_task/rss_reader/requirements.txt') as fp:
+    install_requires = fp.read()
```

Comment:
It is better to manually add them, instead of reading a file here

Reply:
Done